data/Dockerfiles/sogo/Dockerfile

```diff
@@ -8,6 +8,7 @@ ENV GOSU_VERSION 1.9
 # Prerequisites
 RUN apt-get update && apt-get install -y --no-install-recommends \
 		apt-transport-https \
+		bind9-host \
```

Comment:
I think @andryyy decided to switch from bind9 to unbound. Just to be neat we could use the unbound-host package instead of the bind9-host one, or is there a complex reason to use bind9?